src/Module/index.jsx

```diff
@@ -204,7 +198,6 @@ export default class Module extends Component
                     collapsed       : isCollapsible && isCollapsed,
                     error           : hasError,
                     moduleError     : hasModuleError,
-                    subtleHighlight : highlightType === 'subtle',
                     level           : headerLevel
```

Comment:
`level` can be removed from here too, since it’s no longer used in the stylesheet.